### PR TITLE
Simple Calculator

### PR DESCRIPTION
**Pull request by tharun9746**
### **Describe your change:**

- [ x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [ x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [ x] This pull request is all my own work -- I have not plagiarized.
- [x ] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x ] All new Java files are placed inside an existing directory.
- [x ] All filenames are in all uppercase characters with no spaces or dashes.
- [x ] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by tharun9746** [comment]
iam facing some environment issues. Need help with the same.
image

File: Misc/SimpleCalculator.java [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+	JButton plus,clear,close,minus,div,mul,equal;
+	JButton seven,eight,nine,four,five,six,one,two,three,zero,dot;
+	
+	public SimpleCalculator()
+	{
+		jf=new JFrame();                                       /*this is code to create a frame*/
+		jf.setTitle("Calculator");
+		jf.setLayout(null);
+		jf.setSize(600,700);
+		jf.setVisible(true);
```
Comment (ayaankhan98) [inline Misc/SimpleCalculator.java:22]: good to move at the end of ctor

**Review by ayaankhan98 — CHANGES_REQUESTED** [review]
code will not work if you try to use negative numbers.

File: Misc/SimpleCalculator.java (hunk 1 above)
Comment (tharun9746) [reply Misc/SimpleCalculator.java:22]: sorry @ayaankhan98  i didn't get your point.
could you please elaborate the same.
and thanks for helping.

**Pull request closed without merging**